pkgs/development/python-modules/voluptuous-serialize/default.nix

```diff
@@ -1,33 +1,32 @@
-{ lib, stdenv, buildPythonPackage, isPy3k, fetchPypi, voluptuous, pytest }:
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, voluptuous
+, pytestCheckHook
+}:
 
 buildPythonPackage rec  {
   pname = "voluptuous-serialize";
   version = "2.4.0";
 
-  disabled = !isPy3k;
```

Comment:
It doesn't build on Python2 with the following error:
```
Check whether the following modules can be imported: voluptuous_serialize
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/9j34pp0ajq2hzb3hi7zv0l3bpszm4n43-python-2.7.18/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/nix/store/5jv5hiimrkwld75prpxh6a4hf1zps0jy-python2.7-voluptuous-serialize-2.4.0/lib/python2.7/site-packages/voluptuous_serialize/__init__.py", line 17
    def convert(schema, *, custom_serializer=None):
                         ^
SyntaxError: invalid syntax
```

So we should keep this line.

Reply:
Reverted, thanks.